Tasks/AppStoreRelease/auth-helper.ts

```diff
@@ -0,0 +1,10 @@
+export class UserCredentials {
+    username: string;
+    password: string;
+    appSpecificPassword: string;
+    fastlaneSession: string;
+
+    /* tslint:disable:no-empty */
+    public UserCredentials() { }
```

Comment:
Why do we need this empty constructor?

Reply:
Ah, I see it was migrated from old version of this code, but still, do we need it here?

Reply:
Good question, I didn't think about it yet. I will take a look at it, thank you!